pkg/master/master.go

```diff
@@ -955,12 +1000,13 @@ func (m *Master) thirdpartyapi(group, kind, version string) *apiserver.APIGroupV
 
 // experimental returns the resources and codec for the experimental api
 func (m *Master) experimental(c *Config) *apiserver.APIGroupVersion {
-	controllerStorage := expcontrolleretcd.NewStorage(c.DatabaseStorage)
-	autoscalerStorage := horizontalpodautoscaleretcd.NewREST(c.ExpDatabaseStorage)
-	thirdPartyResourceStorage := thirdpartyresourceetcd.NewREST(c.ExpDatabaseStorage)
-	daemonSetStorage, daemonSetStatusStorage := daemonetcd.NewREST(c.ExpDatabaseStorage)
-	deploymentStorage := deploymentetcd.NewStorage(c.ExpDatabaseStorage)
-	jobStorage, jobStatusStorage := jobetcd.NewREST(c.ExpDatabaseStorage)
+	dbClient := func(resource string) storage.Interface { return c.DatabaseStorage.get(resource, true) }
```

Comment:
That's not really true. I already have "get" method in StorageDestionations. Here I'm defining "get("experimental", resource) simply to avoid writing "experimental" in every single line of that code.